### ISSUE TITLE
Error upon pasting rules

### ISSUE THREAD

**Issue by braymp** [CellProfiler/CellProfiler-Analyst#114]
In Classifier, I did the following:
* Selected Classifier > Fast Gentle Boosting
* Selected Advanced > Edit Rules
After pasting some text (from rules I had used successfully in the past), then clicking OK, I got the following error:
```
An error occurred in the program:
AttributeError: 'unicode' object has no attribute 'SetValue'

Traceback (most recent call last):
  File "C:\Trunk\cellProfiler-analyst\cpa\classifier.py", line 1858, in OnRulesEdit
    self.rules_text.Value.SetValue(self.algorithm.ShowModel())
````

**Comment by daviddao** [comment]
Thanks for reporting! I take a look on it later today.

**Comment by daviddao** [comment]
Fixed with 3a5d1e6 . 
```
Error message An error occurred in the program:
AttributeError: 'unicode' object has no attribute 'SetValue'

Traceback (most recent call last):
  File "C:\Trunk\cellProfiler-analyst\cpa\classifier.py", line 1858, in OnRulesEdit
    self.rules_text.Value.SetValue(self.algorithm.ShowModel())
```
resolved by changing ```self.rules_text.Value.SetValue(...)``` to```self.rules_text.SetValue(...)```
